middle_end/flambda/simplify/basic/simplified_named.mli

```diff
@@ -43,7 +43,7 @@ val reachable : Named.t -> t
 
 (** It is an error to pass [Static_consts] to this function. *)
 val reachable_with_known_free_names
-  : find_cost_metrics:(Code_id.t -> Cost_metrics.t)
+  : find_cost_characteristics:(Code_id.t -> Cost_metrics.t * int)
```

Comment:
Please make a new type; it's otherwise not clear what the "int" is.

Reply:
Added a new type and renamed find_cost_characteristics to find_code_characteristics